docs/pt/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# Middleware
+
+Você pode adicionar middleware à suas aplicações **FastAPI**.
+
+Um "middleware" é uma função que manipula cada **request** antes de ser processada por qualquer *operação de caminho* específica. E também cada **response** antes de retorná-la.
+
+* Ele pega cada **request** que chega ao seu aplicativo.
```

Comment:
I would change request to "requisição".

so as not to repeat myself, it applies to all cases in which request does not refer to the Request object.

Reply:
Thank you, I'll fix it